src/main.cpp

```diff
@@ -658,6 +677,8 @@ int main ( int argc, char** argv )
                              bDelayPan,
                              eLicenceType );
 
+            CServerRpc ServerRpc ( &Server, &RpcServer );
```

Comment:
And here, too.

Reply:
Done — CServerRpc is no longer allocated without a CRpcServer.

bbb062387bf35e1f755837329324bb168ae643c8